### DISCUSSION TITLE
Filtering by concatenation of few fields

### DISCUSSION THREAD

**Discussion by itslarin** [TiddlyWiki/TiddlyWiki5#6500] [Q&A]
Hello, there.

Trying to use TW as a task tracker. Having few main types of tiddlers: day, week, sprint, event and task.

All of them have few user fields:

```
day_since, 
month_since, 
year_since, 
day_due, 
month_due, 
year_due.

```
For example,

the task will have fields:

```
day_due="23"
month_due="02"
year_due="2022"
```

the week will have fields:

```
day_since="21"
month_since="02"
year_since="2022"
day_due="27"
month_due="02"
year_due="2022"

```
I do not use date type for purpose - i use xx instead of numbers (cron-like style) for some events - like birthdays and holidays to be repeated every year, so i put xx instead of year number etc. I want to filter tasks to show all tasks, fitting into week duration - i have day, month and year both for start and end of the week andd need to build and expression how to shoow all fitting task on the week card.

Got stuck with searching the solution. I use tiddlywiki 5.2.1. Would appreciate any advices. Thanks in advance.

**Comment by saqimtiaz** [comment]
Hi @itslarin. The Github discussions are meant developers. Questions like yours will get better engagement if you post them at the community forum: https://talk.tiddlywiki.org/

**Reply by itslarin** [reply]
My bad, thank you for reply

**Reply by itslarin** [reply]
please feel free to delete my post as long as i dont see such an option available for me
